### DISCUSSION TITLE
Rename folder "cmd" in V root directory, to avoid Windows cmd.exe clash

### DISCUSSION THREAD

**Discussion by chipnetics** [vlang/v#14656] [Ideas]
The V root directory has a folder named "cmd" which causes grief in Windows when opening a shell prompt by typing "cmd" into the Windows Explorer navigation bar, as Windows instead opens the folder cmd rather than launching cmd.exe.

The only way to circumvent this is to open a different folder in the V root directory, type cmd in the navigation bar, and when the shell prompt launches  `cd ..` back to the V root directory.

**Comment by medvednikov** [comment]
Windows is insane.

**Comment by JalonSolov** [comment]
Told ya we should've dropped Windows support.  :-)

Maybe _only_ support it through cross-compilation, and quit trying to mangle things so V runs on Windows directly?  :-\

**Comment by JalonSolov** [comment]
Seriously, though, a fairly standard layout in Go repos is to have a `cmd` subdir, and I haven't heard anyone complaining about that.

**Comment by chipnetics** [comment]
I think the issue is more prominent here as v.exe is directly at the root and not in a bin folder.  So if your intent is to do something with v.exe and you type cmd in Windows Explorer to get a shell, you open a folder instead.

Quick search on Discord turned up two other confirmations of struggles.

https://discordapp.com/channels/592103645835821068/592294828432424960/796396582919471144

https://discordapp.com/channels/592103645835821068/592115457029308427/932678532394516541

**Comment by Wertzui123** [comment]
I've run into this several times as well, but you can just work around it by typing `cmd.exe` instead.

**Comment by Wertzui123** [comment]
> Told ya we should've dropped Windows support. :-)
> 
> Maybe _only_ support it through cross-compilation, and quit trying to mangle things so V runs on Windows directly? :-\

Even if Windows is a mess, it's still one of the best and most used operating systems out there, so I guess you just have to accept it ¯\_(ツ)_/¯
Also, what's so bad about this particular behavior? The shortcut is really handy, but if there is a "cmd" folder, it should navigate to it, just like it does with other folders.

**Comment by JalonSolov** [comment]
"most used" I agree with. "best"... well, it's almost better than nothing at all...  :-)

The problem is very Windows centric, and as I said, it's not unique to V, except as @chipnetics pointed out - v.exe is at the same level as the subdir, so there can be some issues.

**Comment by dumblob** [comment]
I'm not on Windows but is there any shorter way than typing `cmd.exe`?

Maybe something like `cmd.` or `cmd?` or `cmd*` or `term`/`console` (yeah, user-defined alternative/link to `cmd.exe`) or whatever...

**Comment by dumblob** [comment]
In https://stackoverflow.com/a/40146173 it suggests to press `Shift` + `Menu` and then arrows (or maybe shortcut typing works?) to choose `Open Command Window Here`.

**Edit**: yes, there is shortcut typing support: https://superuser.com/a/205368 ! So this should really be the very fastest way of opening `cmd.exe`.

Might be about the same speed as `Alt` + `D` and typing `cmd` and pressing `Enter`.

Both methods are without mouse involvement (which as I understood is the whole point of this issue).

**Comment by dumblob** [comment]
Or alternatively force to execute a command from the Windows File Explorer navigation bar instead of trying to open a folder first:

`start shell:cmd` (or maybe only `shell:cmd` - but I didn't test this)
